students/synowiec_krzysztof/lesson_05_lists/swap_min_and_max.py

```diff
@@ -0,0 +1,20 @@
+def main():
+    list = [int(x) for x in input().split()]
+    minIndex = 0
+    maxIndex = 0
+    for x in range(1, len(list)):
+        if list[x] > list[maxIndex]:
+            maxIndex = x
+            continue
+        elif list[x] < list[minIndex]:
+            minIndex = x
+
+    temp_max = list[maxIndex]
+    list[maxIndex] = list[minIndex]
+    list[minIndex] = temp_max
+    for x in range(0, len(list)):
```

Comment:
no need for first argument when it's 0. so 
`for x in range(len(list)):` 
does exactly same thing